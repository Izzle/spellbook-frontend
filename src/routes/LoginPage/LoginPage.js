import React, { Component } from 'react';

export default class LoginPage extends Component {
  render() {
    return (
      <section className='LoginPage'>
          <h2>Login will be here!</h2>
      </section>
    )
  }
}