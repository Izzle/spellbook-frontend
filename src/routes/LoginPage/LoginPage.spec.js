import React from 'react';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import LoginPage from './LoginPage';
import { BrowserRouter } from 'react-router-dom';


describe('<LoginPage />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<BrowserRouter><LoginPage /></BrowserRouter>, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    it('renders this UI as expected', () => {
        const tree = renderer.create(<BrowserRouter><LoginPage /></BrowserRouter>).toJSON();
        expect(tree).toMatchSnapshot();
    });    
});