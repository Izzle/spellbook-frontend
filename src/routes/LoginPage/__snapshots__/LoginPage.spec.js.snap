// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginPage /> renders this UI as expected 1`] = `
<section
  className="LoginPage"
>
  <h2>
    Login will be here!
  </h2>
</section>
`;
